fhirstore/fhirstore.py

```diff
@@ -243,7 +243,7 @@ def search(self, resource_type, params):
                 build_simple_query({sub_key: sub_value}) for sub_key, sub_value in params.items()
             ]
             sub_query = {"bool": {"must": inter_query}}
-        query = {"min_score": 0.01, "query": sub_query}
+        query = {"min_score": 0.01, "size": 10000, "query": sub_query}
```

Comment:
I think this should be configurable by the user (in the url)
let's find a solution for this to be fhir compliant

Reply:
The fhir compliant way to do this would be to limit the number of results per page, which would be done in the API, don't you think?

Reply:
100% agree! therefore this function needs to have an optional argument (with a default value) so that the api can specify the number of records that needs to be returned by fhirstore.

Reply:
Okay, I added a result_size argument in the search with a default value of 1000, and unit tests to make sure it does what is asked.

Reply:
Actually I lowered the default value to 100!